src/fragments/lib/auth/native_common/getting_started/common.mdx

```diff
@@ -21,7 +21,14 @@ import flutter2 from "/src/fragments/lib/auth/flutter/getting_started/10_preReq.
 
 <Fragments fragments={{flutter: flutter2}} />
 
-## Configure Auth Category
+## Set Up Backend Resources
+
+To use Authentication with Amplify, you can choose to configure resources through the Amplify CLI (which will automatically create them on the backend for you) or use an existing Amazon Cognito resource in your AWS account.
+
+> Note that before you can add an AWS resource to your application, the application must have the Amplify libraries installed. If you need to perform this step, see [Install Amplify Libraries](/lib/project-setup/create-application#n2-install-amplify-libraries) and ensure you've installed the necessary dependencies.
```

Comment:
I read this as an important note here, should use the `Callout` block instead of the Quote block?

Reply:
Valid point.  I've updated all references to this to be within a `<Callout>` now.  Let me know if this looks better.  

This one was before the more recent updates/latest commit tat dropped the "> Note that..." into the block switcher.    

CC: @abdallahshaban557